static/scss/profile.scss

```diff
@@ -32,6 +32,9 @@
     flex-direction: row;
     color: $lightgrey !important;
     text-transform: uppercase;
+    position: absolute;
+    right: 0;
+    padding-right: 41px;
```

Comment:
again, I don't think these three lines are necessary, I can delete them and the layout is the same